<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="group_student_data">
        <property name="title">
         <string>Анкетные данные</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ФИО</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_full_name">
             <property name="maxLength">
              <number>45</number>
             </property>
             <property name="placeholderText">
              <string>Введите фамилию, имя, отчетсво</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QGroupBox" name="gender">
           <property name="title">
            <string>Пол</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="radio_woman">
              <property name="text">
               <string>Женский</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_man">
              <property name="text">
               <string>Мужской</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Дата рождения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="student_birthday">
             <property name="maximumDateTime">
              <datetime>
               <hour>23</hour>
               <minute>59</minute>
               <second>59</second>
               <year>2080</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="minimumDateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>1950</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="maximumDate">
              <date>
               <year>2080</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>1950</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_medal">
        <property name="text">
         <string>Наличие золотой или серебрянной медали</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Средняя оценка по атестату</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spin_middle_mark">
          <property name="minimum">
           <double>3.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="group_language">
        <property name="title">
         <string>Иностранный язык, который изучали в школе</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="radio_lang_english">
           <property name="text">
            <string>Английский</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_lang_german">
           <property name="text">
            <string>Немецкий</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_lang_french">
           <property name="text">
            <string>Французкий</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_lang_korean">
           <property name="text">
            <string>Корейский</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_lang_chinise">
           <property name="text">
            <string>Китайский</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_lang_japanise">
           <property name="text">
            <string>Японский</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QCheckBox" name="check_dogvor">
          <property name="text">
           <string>Готовность учится на договорной основе</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Стоимость года обучения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_bill">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>12000</number>
          </property>
          <property name="maximum">
           <number>50000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tab_marks">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_one">
         <attribute name="title">
          <string>Предмет 1</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGroupBox" name="group_marks_exam">
            <property name="title">
             <string>Оценки на вступительных экзаменах</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QRadioButton" name="radio_math_one">
                 <property name="text">
                  <string>Математика I</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_math_two">
                 <property name="text">
                  <string>Математика II</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_math">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QRadioButton" name="radio_phys_one">
                 <property name="text">
                  <string>Физика</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_phys_two">
                 <property name="text">
                  <string>Физика II</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_phis">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QRadioButton" name="radio_rus">
                 <property name="text">
                  <string>Русский язык</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_rus">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_two">
         <attribute name="title">
          <string>Предмет 2</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="group_marks_exam_2">
            <property name="title">
             <string>Оценки на вступительных экзаменах</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QRadioButton" name="radio_math_one_2">
                 <property name="text">
                  <string>Математика I</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_math_two_2">
                 <property name="text">
                  <string>Математика II</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_math_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QRadioButton" name="radio_phys_one_2">
                 <property name="text">
                  <string>Физика</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_phys_two_2">
                 <property name="text">
                  <string>Физика II</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_phis_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QRadioButton" name="radio_rus_2">
                 <property name="text">
                  <string>Русский язык</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_rus_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_three">
         <attribute name="title">
          <string>Предмет 3</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QGroupBox" name="group_marks_exam_3">
            <property name="title">
             <string>Оценки на вступительных экзаменах</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QRadioButton" name="radio_math_one_3">
                 <property name="text">
                  <string>Математика I</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_math_two_3">
                 <property name="text">
                  <string>Математика II</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_math_3">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QRadioButton" name="radio_phys_one_3">
                 <property name="text">
                  <string>Физика</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radio_phys_3">
                 <property name="text">
                  <string>Физика II</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_phis_3">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QRadioButton" name="radio_rus_3">
                 <property name="text">
                  <string>Русский язык</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_rus_3">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QTableWidget" name="table_students">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>ФИО</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Средний балл по вступительным экзаменам</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="btn_create">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_fill">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Заполнить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_save">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_delete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="create_file"/>
    <addaction name="open_from_file"/>
    <addaction name="safe_to_file"/>
    <addaction name="save_as_to_file"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="create_file">
   <property name="text">
    <string> Создать базу данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open_from_file">
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="safe_to_file">
   <property name="text">
    <string> Сохранить базу данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="save_as_to_file">
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
